users/templates/user_site/home.html

```diff
@@ -0,0 +1,85 @@
+{% extends "base.html" %}
+
+{% block head-content %}
+    <meta charset="utf-8"/>
+    <title>stocks</title>
```

Comment:
IMHO i think our main page title should be lets-meet

Reply:
Almost missed that one, 
thank you for noticing 👍